microshift_storage/expanding-persistent-volumes-microshift.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="expanding-persistent-volumes-microshift"]
+= Expanding persistent volumes for MicroShift
+include::_attributes/common-attributes.adoc[]
+:context: expanding-persistent-volumes-microshift
+
```

Comment:
I'd normally point out to have an intro, but again I see it's like that in the original storage one. So letting slide, but again just pointing out for posterity. I won't point out any further ones either.